config/samples/perf_v1alpha1_iperf3.yaml

```diff
@@ -33,6 +33,17 @@ spec:
       # nodeName: null
     hostNetwork: false
     
-  # clientConfiguration: null
+  clientConfiguration:
```

Comment:
I don't like this. As a user, how should I come up with this structure? Of course, I can find it in the Kubernetes API, but imho it would be so much easier having simple flags for this. E.g. "iperfServerAntiAffinity: true". Maybe beside the flags you can add the option to customize anything, but having this is a requirement just really doesn't seem user friendly to me.

Reply:
It is described in the CRD to every detail:
```
            Affinity:
              Description:  Affinity is a group of affinity scheduling rules.
              Properties:
                Node Affinity:
                  Description:  Node affinity is a group of node affinity scheduling rules.
                  Properties:
```

We are using kubernetes native types, hence we could point the user to the necessary document in each section:
https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity

Documenting it in the sample is not the right place imho. We should start creating user docs which contains all these details.

Reply:
regarding simplification with one variable: it provides less flexibility and i'd rather not define the common use-cases at this time (how granular we need?)